client/settings/connect-stripe-account/index.js

```diff
@@ -48,94 +47,63 @@ const ButtonWrapper = styled.div`
 `;
 
 const ConnectStripeAccount = ( { oauthUrl } ) => {
-	// @todo - deconstruct modalType and setModalType from useModalType custom hook
-	const [ modalType, setModalType ] = useState( '' );
-	const handleModalDismiss = () => {
-		setModalType( '' );
-	};
-
 	const handleCreateOrConnectAccount = () => {
 		recordEvent( 'wcstripe_create_or_connect_account_click', {} );
 		window.location.assign( oauthUrl );
 	};
 
-	const handleEnterAccountKeys = () => {
-		recordEvent( 'wcstripe_enter_account_keys_click', {} );
-		setModalType( 'live' );
-	};
-
 	return (
-		<>
```

Comment:
There are a lot of changes here, however, all I've done is remove the scenario were `oauthUrl` is empty and it use to display a

| Before | After |
|--------|--------|
| <img width="400" alt="Screenshot 2024-07-17 at 3 24 08 PM" src="https://github.com/user-attachments/assets/16cc27ae-1818-485b-bd95-a0c1fc4c1942"> | <img width="500" alt="Screenshot 2024-07-18 at 1 55 38 PM" src="https://github.com/user-attachments/assets/f9ede0fc-71fa-4c05-9d27-31dd52736858"> |